py/src/nupic/examples/mnist.py

```diff
@@ -87,16 +87,16 @@ def encode(self, image):
 
 
 default_parameters = {
-    'boostStrength': 2.5,
-    'columnDimensions': (16000,),
-    'dutyCyclePeriod': 1402,
-    'localAreaDensity': 0.024,
-    'minPctOverlapDutyCycle': 0.001,
-    'potentialPct': 0.2,
-    'stimulusThreshold': 6,
-    'synPermActiveInc': 0.01,
-    'synPermConnected': 0.422,
-    'synPermInactiveDec': 0.005
+    'boostStrength': 7.80643753517375,
+    'columnDimensions': (35415,),
```

Comment:
OT: I wonder how much the scores differ? It's said HTM is/should be more resistant to param changes. These seem like a rather big difference.

Reply:
IIRC accuracy improved from about 90% to 96%.